tools/scenario-player/scenario_player/tasks/channels.py

```diff
@@ -10,13 +10,17 @@ class OpenChannelTask(RaidenAPIActionTask):
 
     @property
     def _request_params(self):
-        params = dict(
-            token_address=self._runner.token_address,
-            partner_address=self._runner.node_to_address[
+        if isinstance(self._config['to'], str) and len(self._config['to']) == 42:
```

Comment:
I'd change the second condition to `eth_utils.is_checksum_address(self._config['to'])`

Reply:
AFAIK we intentionally want to allow invalid addresses for testing purposes

Reply:
But `is_normalized_address` expects all lowercase doesn't it?

Reply:
True again ;)

Reply:
Shouldn't we make different tasks for this? We want to allow tests with invalid addresses, but for that we don't need to make the ones that use valid address more error prone.

Reply:
@hackaugusto For the 'normal' ones we dont input addresses at all but rather use an int index into the known nodes list.

Reply:
No, but it should be possible to test for example non-checksummed addresses

Reply:
> but it should be possible to test for example non-checksummed addresses

I'm not saying otherwise, I'm saying that this line:

`- transfer: {from: 0, to: '0x3c4B735477206e2f656c7966485E78795e592372', amount: 1}`

Tells nothing about if the address is valid/if the transfer is expected to succeed or not. And I believe using a different name can be more informative, e.g.:

`- transfer_invalid: {from: 0, to: '0x3c4B735477206e2f656c7966485E78795e592372', amount: 1}`

Reply:
That’s not necessary. All API related tasks have an `expected_http_status` parameter that defaults to `2..` (regex)